client/state/wordads/earnings/selectors.js

```diff
@@ -0,0 +1,32 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+export function isRequestingWordadsEarnings( state, siteId ) {
+	return !! state.wordads.earnings.fetchingItems[ siteId ];
+}
+
+/**
+ * Returns earnings object for a siteId
+ * @param   {Object} state  Global State
+ * @param   {Number} siteId Site Id
+ * @returns {Object}        WordAds Error
+ */
+export function getWordAdsEarnings( state, siteId ) {
+	return state.wordads.earnings.items[ siteId ];
+}
+
+/**
+ * Sanitizes site object and returns object if the WordAds earnings request was successful
+ * @param   {Object} state  Global State
+ * @param   {Object} site   Site
+ * @returns {Object}        WordAds Error
+ */
+export function getWordAdsEarningsForSite( state, site ) {
```

Comment:
Is this selector really needed? It would be better to have just one `getWordAdsEarnings( state, siteId )` and use it everywhere.